ibc-clients/README.md

```diff
@@ -25,7 +25,7 @@ IBC light clients:
 
 - [ibc-client-tendermint-types](./ics07-tendermint/types): Data Structures
 - [ibc-client-tendermint](./ics07-tendermint): Implementation
-- [ibc-client-tendermint-cw](https://github.com/informalsystems/cosmwasm-ibc/ibc-clients/ics07-tendermint): CosmWasm
+- [ibc-client-tendermint-cw](https://github.com/cosmos/sp1-ics07-tendermint): CosmWasm
```

Comment:
The correct link is: https://github.com/informalsystems/cosmwasm-ibc/tree/main/ibc-clients/ics07-tendermint